### PR TITLE
Update README.md with conda install instructions

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
 <p align="center">
-<img width=50% src="https://evalml-web-images.s3.amazonaws.com/evalml_horizontal.svg" alt="Featuretools" />
+<img width=50% src="https://evalml-web-images.s3.amazonaws.com/evalml_horizontal.svg" alt="EvalML" />
```
Comment (freddyaboulton) [inline README.md:2]: 👍

**Review by freddyaboulton — APPROVED** [review]
Thank you @gsheni !!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+    <a href="https://evalml.alteryx.com/en/latest/?badge=stable" target="_blank">
+        <img src="https://readthedocs.com/projects/feature-labs-inc-evalml/badge/?version=stable" alt="Documentation Status" />
+    </a>
+    <a href="https://badge.fury.io/py/evalml" target="_blank">
+        <img src="https://badge.fury.io/py/evalml.svg?maxAge=2592000" alt="PyPI Version" />
+    </a>
+    <a href="https://anaconda.org/conda-forge/evalml" target="_blank">
+        <img src="https://anaconda.org/conda-forge/evalml/badges/version.svg" alt="Anaconda Version" />
+    </a>
+    <a href="https://pepy.tech/project/evalml" target="_blank">
```
[the remark this answers is not in the document]
Comment (gsheni) [reply README.md:21]: Not sure. One uses shields.io and the other uses pepy.tech

[the remark this answers is not in the document]
Comment (freddyaboulton) [reply README.md:21]: Well these are higher so that's good 😂

[the remark this answers is not in the document]
Comment (dsherry) [reply README.md:21]: Yeah weird, they must gather data in different ways

**Pull request merged**